go/logic/orchestrator.go

```diff
@@ -70,7 +70,7 @@ var isElectedNode int64 = 0
 
 var recentDiscoveryOperationKeys *cache.Cache
 var pseudoGTIDPublishCache = cache.New(time.Minute, time.Second)
-var kvFoundCache = cache.New(10*time.Minute, time.Minute)
+var kvFoundCache = cache.New(10*time.Second, time.Second)
```

Comment:
I'm not sure how changing it so drastically affects things. Performance? Network traffic? 
Maybe better to leave the default as it was and introduce a new config variable and use it like `instancePollSecondsDuration()` is used?

Reply:
Indeed it will cause extra lookups to the k/v which might be problematic for some environments with lots of servers. I agree about adding a new config variable